suricata.yaml.in

```diff
@@ -469,6 +469,8 @@ af-packet:
     #rollover: yes
     # To use the ring feature of AF_PACKET, set 'use-mmap' to yes
     use-mmap: yes
+    # Use tpacket_v3, capture mode, only active if user-mmap is true
+    #tpacket-v3: true
```

Comment:
We can detect this at compile time right? So default to v3 if available, use v2 otherwise or if user specifies it explicitly.